code/weapon/swarm.cpp

```diff
@@ -35,11 +35,8 @@
 #define TURRET_SWARM_VALIDITY_CHECKTIME	5000	// number of ms between checks on turret_swam_info checks
 
 #define SWARM_USED						(1<<0)
```

Comment:
It looks like `SWARM_USED` can also be removed, if I'm reading things right.

Reply:
It is still used by turret swarm handling unfortunately.